### ISSUE TITLE
Everything breaks when switching from Spongemixins 1.4.0 to Gas Station

### ISSUE THREAD

**Issue by brandyyn** [FalsePattern/FalseTweaks#73]
After swapping to gas station, I can't launch my game, but obviously I want to use falsetweaks and the updated versions of other mods like archaicfix. I've tried gas station 0.4 and 0.3.5(because I saw makamys using it) here's the log while using 0.3.5:
[latest.log](https://github.com/FalsePattern/FalseTweaks/files/10095529/latest.log)
and here's the log using 0.4
[latest.log](https://github.com/FalsePattern/FalseTweaks/files/10095546/latest.log)
notably, here you can see it's having a problem with @makamys satchels mod too?
between these i disabled the snowball fix from foamfix error the first log seemed to be having trouble with

**Comment by brandyyn** [comment]
also seems to conflict with @makamys coretweaks
[latest.log](https://github.com/FalsePattern/FalseTweaks/files/10095567/latest.log)

**Comment by brandyyn** [comment]
disabling the swimfix from coretweaks got rid of that issue but now there's an error with bugtorch
[latest.log](https://github.com/FalsePattern/FalseTweaks/files/10095570/latest.log)

**Comment by brandyyn** [comment]
ignore the 2 replies i deleted, i forgot to add gas station back for those reports. 
here's another one that seems different with 0.3.5
[latest.log](https://github.com/FalsePattern/FalseTweaks/files/10095585/latest.log)

**Comment by brandyyn** [comment]
removed every mod i saw an error with and now there are none and i have no idea what to do next
[latest.log](https://github.com/FalsePattern/FalseTweaks/files/10095594/latest.log)

**Comment by makamys** [comment]
That last stack trace looks similar to the one you posted a while ago when you tried to update from SpongeMixins 1.4.0 to 1.5.0 (Mixin 0.7.11 -> 0.7.12). If I understand correctly, Mixin became stricter about verifying the superclasses of mixin targets in 0.7.12. Which is a problem because some mod (Starminer iirc) changes the superclass of EntityPlayer. Try what happens when you remove Starminer.

**Comment by brandyyn** [comment]
in the last stacktrace, i removed alfheim and asjcore. this one i removed those 2 and i removed starminer while using 0.3.5 and the game actually launches, so what do i do? because these are mods I want to play with. It doesn't work with alfheim(+asjcore) or starminer above spongemixins 1.4.0- what can i do?

**Comment by brandyyn** [comment]
i just realized i put this issue in the wrong place, my bad- but also after installing falsetweaks there are glitchy overlapping textures https://i.imgur.com/MC83mF2.png

**Comment by makamys** [comment]
Alfheim seems to be unrelated. I can reproduce the last crash (the `Delegate constructor lookup failed for @Inject target on satchels.mixin.json:MixinEntityPlayer` one) with just Starminer + Satchels + GasStation.


Full mod list:
```
00gasstation-mc1.7.10-0.4.0.jar
CodeChickenCore-1.7.10-1.0.7.48-universal.jar
Satchels-1.7.10-1.0.7.jar
Starminer1710-0.9.9.jar
1.7.10/CodeChickenLib-1.7.10-1.1.3.141-universal.jar
```


> what can i do?

You can keep using Mixin 0.7.11 (SpongeMixins 1.4.0) until a fix is discovered. The only mods that need 0.8 right now to my knowledge are the recent versions of LagGogglesLegacy, and GTNH's coremods.

> after installing falsetweaks there are glitchy overlapping textures

I think you should open a separate issue for that one.

**Comment by tyrael93** [comment]
> The only mods that need 0.8 right now to my knowledge are the recent versions of LagGogglesLegacy, and GTNH's coremods.

It's worth a mention that you can actually change that to work with 0.7.11 manually.

**Comment by makamys** [comment]
Right, you can force mods to work with Mixin 0.7.11 if you change `minVersion` to 0.7.11 in the mixin jsons in the root of the mod jar (usually named `mixins.<mod_name>.json`).

But if you do this, please rename the jar to mark that it has been modified (for example, add `+mixin0.7` to the end), so we can tell you're using a modified version in crash logs and such.

**Comment by makamys** [comment]
Nope! They work with 0.7. And I think LagGoggles does too with the little trick above. GTNH's new stuff actually depends on GTHMixins (Mixin 0.8) though.

**Comment by brandyyn** [comment]
You're right, they do! I just tested it :D thank you! I'll stick to 1.4.0 then for now
